contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -177,11 +177,11 @@ contract VaultHub is PausableUntilWithRoles {
         uint256 reserveRatioBP = socket.reserveRatioBP;
         uint256 maxMintableRatio = (TOTAL_BASIS_POINTS - reserveRatioBP);
         uint256 valuation = IStakingVault(_vault).valuation();
+        bool isHealthy = isVaultHealthy(_vault);
 
-        // to avoid revert below
-        if (mintedStETH * TOTAL_BASIS_POINTS < valuation * maxMintableRatio) {
-            // return MAX_UINT_256
-            return type(uint256).max;
+        // Health vault do not need to rebalance
+        if (isHealthy) {
```

Comment:
let's structure this func this way:
- if healthy early `return 0`
- if 'unrebalanceable' early return +inf (i.e., bad debt): `mintedStETH > valuation`
- calculate otherwise